integration-libs/opf/base/components/opf-quick-buy/apple-pay/observable/apple-pay-observable.factory.ts

```diff
@@ -72,7 +76,10 @@ export class ApplePayObservableFactory {
         });
       }
 
-      if (config.shippingMethodSelected) {
+      if (
+        config.shippingMethodSelected &&
+        config.request.shippingType !== ApplePayShippingType.STORE_PICKUP
+      ) {
```

Comment:
We duplicate this code two times, maybe is it worth to move it into some function that do such check?